app/controllers/text_extraction_controller.rb

```diff
@@ -6,6 +6,8 @@ class TextExtractionController < ApplicationController
   def new; end
 
   def create
+    return redirect_to solr_document_path(@cocina_object.externalIdentifier), flash: { error: 'Object is not open' } unless VersionService.open?(druid: @cocina_object.externalIdentifier)
+
```

Comment:
we disable the button if not open, this just guards against on the server side too